project/AppDependencies.scala

```diff
@@ -61,7 +61,7 @@ object AppDependencies {
         "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.4" % scope,
         "org.scalaj" %% "scalaj-http" % scalaJVersion % scope,
         "org.mockito" % "mockito-core" % mockitoVersion % scope,
-        "com.github.fge" % "json-schema-validator" % "2.2.14" % scope,
+        "com.github.fge" % "json-schema-validator" % "2.2.6" % scope,
```

Comment:
No idea why this worked, as I can't resolve 2.2.14.  Happy to revert if someone can tell me why and fix it.